src/Command/CodeStudio/CodeStudioCommandTrait.php

```diff
@@ -183,7 +183,7 @@ protected function authenticateWithGitLab(): void {
    *
    * @return array
    */
-  protected function determineGitLabProject(ApplicationResponse $cloud_application): array {
+  protected function determineGitLabProject(ApplicationResponse $cloud_application, Bool $create_new_cs_project = TRUE): array {
```

Comment:
The only other code that calls this method is `codestudio:wizard` right? Please change that to pass `TRUE` explicitly and remove the default parameter value here.